test/new-e2e/tests/agent-platform/package-signing/package_signing_test.go

```diff
@@ -86,7 +85,6 @@ func TestPackageSigningComponent(t *testing.T) {
 			&packageSigningTestSuite{osName: platform},
 			e2e.WithProvisioner(awshost.ProvisionerNoFakeIntake(
 				awshost.WithEC2InstanceOptions(ec2.WithAMI(ami, osDesc, osDesc.Architecture)),
-				awshost.WithAgentOptions(agentparams.WithPipeline(os.Getenv("CI_PIPELINE_ID"))),
```

Comment:
Note for the review: This is not required, default behavior of the framework is to install the agent with the version built in the pipeline